chromaui/chromatic-e2e
# Create constructable stylesheet reproductions and internal tests

Issue: #AP-4986

## What Changed



This adds a group of test server pages that use constructible stylesheets, along with e2e tests of these files in both Cypress and Playwright. These tests will be used to verify that styles from constructable stylesheets are rendered properly, once support is added for that feature.

## How to test



1. Run `yarn test:cypress`
2. Run `yarn archive-storybook:cypress`
3. Notice that no colorful styles are added to any of the new `constructable-stylesheets` stories
4. Run `yarn test:playwright`
5. Run `yarn archive-storybook:playwright`
6. Notice that no colorful styles are added to any of the new `constructable-stylesheets` stories
